docs/concepts/assets/accounts.md

```diff
@@ -24,6 +24,35 @@ Any assets that are in a general account can be withdrawn or transferred.
 **[REST](../../api/rest/data-v2/trading-data-service-get-party.api.mdx)**: Use the API to check your public key for the accounts your assets are allocated to.
 :::
 
+## Vault
+A *vault* holds assets that have been contributed by one or more parties to be used by another party for trading. The assets in a vault can only be used to enter and exit positions. Vaults have general and margin accounts associated with them.
+
+Contributors to a vault have a share of the value in the vault, based on how much their deposited amount contributed to its original value.
+
+The vault account manager - the party that created the account - can charge fees on account withdrawals if they choose to.
+
+Each vault account handles one asset.
+
+### Contribute to a vault
+To deposit assets into a vault, a participant will need to submit a `depositIntoVault` transaction which includes the vault ID (account ID), and the amount to contribute.
+
+### Redeem from a vault
```

Comment:
I think you want to add information on how to actually withdraw. This below explains how the mechanics of redemption works on the vault side, but not that I need to send a transaction saying how much I want to withdraw etc.

Reply:
also worth mentioning the cutoff that is added to the withdrawal time.